index.html

```diff
@@ -142,7 +142,7 @@ <h2 class="flush-top font-size-base text-gray-light">
       </h2>
 
       {% if managed_features.size > 0 %}
-        {% assign features = managed_features | slice: 0, 4 %}
+        {% assign features = managed_features | slice: 0, 100 %}
```

Comment:
would this work?
`{% assign features = feature_obj.docs | slice: 0, 100 %}`

Then you could remove line 133